addons/mail/static/src/components/message_context_menu/message_context_menu.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageContextMenu" owl="1">
+        <t t-if="MessageContextMenu">
+            <div class="o_EmojiPickerView d-flex flex-column" t-attf-class="{{ className }}" t-ref="root">
+                <EmojiPickerHeaderView record="emojiPickerView.headerView"/>
+                <div>
+                    <div>
+                        <h1>Tell Something</h1>
+                    </div>
+                </div>
+            </div>
```

Comment:
I suppose this is still work-in-progress.

Just a comment here to do changes to make it work (and few cleaning):

- Don't need `t-if="MessageContextMenu"`. Here it's always falsy (this equivalent to `this.MessageContextMenu` where `this` is the component. Its value is `undefined`, which is falsy in JS, hence it's not rendering children.
- `o_EmojiPickerView` => `o_MessageContextMenu` (btw these classnames, what we call "structural" classnames in Discuss team, are classes to specifically target elements in Discuss code. It's a flavour of BEM principles https://getbem.com with `o_<ComponentName>_<partName>` and `o-<modifierName>`)